internal/commands/uninstall.go

```diff
@@ -41,6 +47,14 @@ func runUninstall(dockerCli command.Cli, installationName string, opts credentia
 	if err != nil {
 		return err
 	}
+	if opts.force {
+		defer func() {
+			if err := installationStore.Delete(installationName); err != nil {
+				fmt.Fprintf(os.Stderr, "failed to force deletion of installation %q: %s\n", installationName, err)
+			}
+			fmt.Fprintf(os.Stderr, "deletion forced for installation %q\n", installationName)
```

Comment:
It seems like this would still happen be printed even if the original `installationStore.Delete` in the mainline bit of the function succeeded? Should this be using the `retErr` pattern to only run if things actually failed?

Reply:
Right 👍 Should be fixed